src/Database/PG/Query/Class.hs

```diff
@@ -209,16 +209,20 @@ instance FromRow a => FromRes [a] where
     throwError "Expecting data. Instead, status is 'CommandOk'"
   fromRes (ResultOkData res) = do
     rm <- liftIO $ buildMat res
-    ExceptT $ return $ fmap V.toList $
-      sequence $ V.map fromRow rm
+    ExceptT $
+      return $
+        fmap V.toList $
+          sequence $ V.map fromRow rm
```

Comment:
Yeah, this is definitely an example of something that `ormolu` makes slightly more unpleasant.

I don’t think it makes sense to change in this PR, but we might want to adopt something in our style guide that encourages `foo . bar $ baz quux` which shouldn’t have this issue.

Reply:
I tweaked some of these now, e.g. putting this one on a single line.